rest/route/project.go

```diff
@@ -979,8 +979,8 @@ func (p *GetPatchTriggerAliasHandler) Parse(ctx context.Context, r *http.Request
 }
 
 func (p *GetPatchTriggerAliasHandler) Run(ctx context.Context) gimlet.Responder {
-	proj, err := dbModel.FindOneProjectRef(p.projectID)
-	if err != nil {
+	proj, err := dbModel.FindMergedProjectRef(p.projectID)
+	if err != nil || proj == nil {
```

Comment:
Maybe it would be preferable to case errors separately on err != nil and proj == nil with different messages to be more clear.